tests/album-delete.test.js

```diff
@@ -0,0 +1,63 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Delete Album', () => {
+  let album;
+  let artist;
+  let singleArtistId;
+  beforeEach(async () => {
```

Comment:
Id leave a clear line between the variable creation and the start of your `beforeEach`:
```
  let singleArtistId;
  
  beforeEach(async () => {
```